infra/aws/terraform/accounts/main.tf

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2022 The CNCF Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+variable "alert-emails" {
+  type    = list(string)
+  default = ["jeefy@cncf.io"]
```

Comment:
Could we add `projects@cncf.io` as well? I don't want to be a single point of failure. :)

Reply:
Absolutely, let's fix that! Added in 7833134